code/game/machinery/doors/firedoor.dm

```diff
@@ -132,7 +132,7 @@
 	user.visible_message("<span class='notice'>[user] starts [welded ? "unwelding" : "welding"] [src].</span>", "<span class='notice'>You start welding [src].</span>")
 	if(W.use_tool(src, user, 40, volume=50))
 		welded = !welded
-		to_chat(user, "<span class='danger'>[user] [welded?"welds":"unwelds"] [src].</span>", "<span class='notice'>You [welded ? "weld" : "unweld"] [src].</span>")
+		to_chat(user, "<span class='danger'>[user] [welded?"welds":"unwelded"] [src].</span>", "<span class='notice'>You [welded ? "weld" : "unweld"] [src].</span>")

```

Comment:
Nitpick: separates.

Unweld is not a word though it does explain what a player is doing.

Reply:
"unweld" is fine, but using a different tense in either message seems weird.